db/migrate/20240527183938_create_bookings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBookings < ActiveRecord::Migration[7.1]
+  def change
+    create_table :bookings do |t|
```

Comment:
I actually like that you decided to call this table "bookings". Also acceptable would be the standard convention of calling it "guest_rooms", but bookings makes sense!